src/main.cpp

```diff
@@ -166,7 +163,17 @@ int main(int argc, char *argv[]) {
     // LauncherV8Platform v8Platform;
     // v8::V8::InitializePlatform((v8::Platform*) &v8Platform);
     // v8::V8::Initialize();
-    instance->initializeServer(minecraftApp, whitelist, &permissionsFile, &pathmgr, idleTimeout, props.worldDir.get(), props.worldName.get(), props.motd.get(), levelSettings, props.viewDistance, true, { props.port, props.portV6, props.maxPlayers }, props.onlineMode, {}, "normal", *mce::UUID::EMPTY, eventing, resourcePackRepo, ctm, *resourcePackManager, createLevelStorageFunc, pathmgr.getWorldsPath(), nullptr, mcpe::string(), mcpe::string(), std::move(eduOptions), resourcePackManager, []() {
```

Comment:
This command alters pathmgr.getWorldsPath()'s returnvalue from "worlds/" to "/games/com.mojang/minecraftWorlds"